contracts/contracts/FeeSub.sol

```diff
@@ -100,6 +100,29 @@ contract FeeSub is Ownable {
         token.safeTransfer(recipient, amount);
     }
 
+    function tokenAmountCanBeSubsidized(
```

Comment:
only thing i can see is maybe there would be a race condition here

Reply:
so two users opened and funds were there in contract initiated the tx but one used the remaining fund but other couldnt

Reply:
only one of the txs will succeed. It can happen that there is enough money in the contract when performing the check, but by the time the txs are included in a block the money was exhausted by a different tx. That is however something we cannot do anything about.